sys/include/crypto/ciphers.h

```diff
@@ -62,13 +62,17 @@ extern "C" {
 #endif
 
 /**
- * error codes
+ * return codes
```

Comment:
(A bit unrelated to your PR since you didn't introduced it, but this comment creates an unwanted/misleading documentation: http://riot-os.org/api/ciphers_8h.html#a929ec194f91a3c680c52b70a149a89ac)